### ISSUE TITLE
Hebrew translation

### ISSUE THREAD

**Issue by GreenLunar** [bartTC/dpaste#77]
Attached Hebrew translation
[dpaste.po.gz](https://github.com/bartTC/dpaste/files/310230/dpaste.po.gz)

**Comment by bartTC** [comment]
Thank you. Quite a lot of strings are missing, is that intentional?

**Comment by GreenLunar** [comment]
I am not sure how to translate these 9 strings.  I will think about it and try afterwards.

**Comment by GreenLunar** [comment]
Until you push this translation, please apply it at https://dpaste.de/ so I can test it.

**Comment by bartTC** [comment]
Thank you @GreenLunar I appreciate your work here but I strongly suggest you setup your own local devlepment build and do *the usual way*. This is documented http://dpaste.readthedocs.io/en/latest/installation.html

I mean, come on, we don't test something with a live, production instance. 😏

**Comment by GreenLunar** [comment]
Translation has been tested [dpaste.po.gz](https://github.com/bartTC/dpaste/files/343578/dpaste.po.gz)

We do need to adopt dpaste for RTL.

**Comment by GreenLunar** [comment]
bump!

**Comment by GreenLunar** [comment]
I do not understand, why this issue has not yet been acted upon?

**Comment by bartTC** [comment]
This is in the codebase but I'm having issues with the RTL of the site. This will be fully implemented once the redesign is done.

**Comment by GreenLunar** [comment]
Just set `direction: rtl;` and always set `direction: rtl;` for text input.

**Comment by bartTC** [comment]
> Just set direction: rtl; and always set direction: rtl; for text input.

Well it's not that easy and thats the problem. The page content is RTL but all of the code is always LTR, so code and input elements must remain LTR. 

![bildschirmfoto 2016-12-21 um 22 22 53](https://cloud.githubusercontent.com/assets/1896/21406452/071b931e-c7cc-11e6-90db-12312c296aa6.png)

![bildschirmfoto 2016-12-21 um 22 21 19](https://cloud.githubusercontent.com/assets/1896/21406442/fa6e2a1e-c7cb-11e6-9b2e-85c2bdf9492e.png)

I don't have an easy way to set this since these elements are scattered across templates. 

Unless somebody wants to implement this, I mark this ticket as 'wontfix' for now. I'll keep this in mind on the redesign/bootstrap removal.

**Comment by GreenLunar** [comment]
Please upload an example page (HTML/CSS), and I will try to send a fix.

**Comment by GreenLunar** [comment]
Try this code.
```
body {
    direction: rtl;
}

#id_content {
    direction: ltr;
}
```
